composer.json

```diff
@@ -17,7 +17,8 @@
     },
     "autoload": {
         "classmap": [
-            "database"
+            "database",
+            "app/Models"
```

Comment:
This shouldn't be necessary since the models are all loaded in the PSR-4 `Rogue\` namespace defined below.